pybaseball/teamid_lookup.py

```diff
@@ -33,6 +33,26 @@ def team_ids(season: Optional[int] = None, league: str = 'ALL') -> pd.DataFrame:
     return fg_team_data
 
 
+def mlb_com_team_id(teamName = None):
```

Comment:
Nitpicky: I think `mlb_com_team_id` is going to be hard to remember. Prefer `mlb_team_id` or `mlbam_team_id` personally

Reply:
Agree, changing the name to `mlb_team_id` will give a more intuitive name to the function.